commands/releases/rollback.js

```diff
@@ -14,12 +14,25 @@ function * run (context, heroku) {
     release = yield releases.FindRelease(heroku, context.app, (releases) => releases.filter((r) => r.status === 'succeeded')[1])
   }
 
+  let latest
   yield cli.action(`Rolling back ${cli.color.app(context.app)} to ${cli.color.green('v' + release.version)}`, {success: false}, co(function * () {
-    let latest = yield heroku.post(`/apps/${context.app}/releases`, {body: {release: release.id}})
+    latest = yield heroku.post(`/apps/${context.app}/releases`, {body: {release: release.id}})
+
     cli.action.done(`done, ${cli.color.green('v' + latest.version)}`)
     cli.warn(`Rollback affects code and config vars; it doesn't add or remove addons.
 To undo, run: ${cli.color.cmd('heroku rollback v' + (latest.version - 1))}`)
   }))
+
+  if (latest.output_stream_url) {
+    cli.log('Running release command...')
+    yield new Promise(function (resolve, reject) {
+      let stream = cli.got.stream(release.output_stream_url)
+      stream.on('error', reject)
+      stream.on('end', resolve)
+      let piped = stream.pipe(process.stdout)
+      piped.on('error', reject)
+    })
+  }
```

Comment:
@dmathieu could you DRY the streaming into `lib` so that `releases:output` and rollback could share the same implementation?